### ISSUE TITLE
Add configuration option to show Guake on the system taskbar

### ISSUE THREAD

**Issue by ferenczy** [Guake/guake#2214]
I would greatly appreciate having a **configuration option to enable the showing of the Guake window on the system taskbar** added, which, when enabled, should also cause it to appear on the <kbd>Alt</kbd>+<kbd>Tab</kbd> list.

# Motivation

I often need to switch between Guake and some other window back and forth repeatedly without Guake losing the focus so I can still see its contents. Currently, to switch back from Guake to the previously focused window without having to use the mouse, I have to press <kbd>Alt</kbd>+<kbd>Tab</kbd>, keep <kbd>Alt</kbd> pressed, and press <kbd>Alt</kbd>+<kbd>Shift</kbd>+<kbd>Tab</kbd> to move the focus where I need, which is super annoying.

It's the same issue as described in the issue https://github.com/Guake/guake/issues/2181. I just propose a different solution.

# Note

The Feature request issue template suggests using a website called FeatHub (https://feathub.com/Guake/guake) for reporting feature requests, but its contents look highly suspicious, so it should be probably reviewed and possibly removed.

**Comment by novaTopFlex** [comment]
On my version of Guake (unsure which version exactly), I do have an icon on the systray, but my more recent issue revolves around the icon disappearing and being replaced with a warning icon. However, the general purpose other than the icon remains in my setup.

**Comment by novaTopFlex** [comment]
> # Note
> The Feature request issue template suggests using a website called FeatHub (https://feathub.com/Guake/guake) for reporting feature requests, but its contents look highly suspicious, so it should be probably reviewed and possibly removed.

Yes, the website now has Indonesian contents that are totally unrelated to Guake or to the supposed "FeatHub" platform.
